src/main/java/seedu/address/ui/JinglePlayer.java

```diff
@@ -0,0 +1,24 @@
+package seedu.address.ui;
```

Comment:
Not sure if its because of my system but the jingle seems to get cut off halfway through when playing on startup.